webui/react/src/services/apiConfig.ts

```diff
@@ -61,21 +61,7 @@ export const login: HttpApi<Credentials, void> = {
   postProcess: noOp,
 };
 
-export const getCurrentUser: HttpApi<EmptyParams, DetailedUser> = {
```

Comment:
@hkang1 shall we keep the old endpoint around? what about the decoder and io types